content/en/blog/_posts/2024-05-01-secret-sync-controller-alpha.md

```diff
@@ -0,0 +1,62 @@
+
+## What is the motivation for this feature?
+
+Many production-scale Kubernetes solutions use secrets stored in an external secret store such as a cloud provider secret management system. These solutions face the same problem: securely accessing those secrets within a Kubernetes cluster. Leveraging a secret store CSI driver allows access to secrets while a cluster is online; however, until now, the secret store CSI Driver has yet to provide a unified solution for access to external secrets while offline. The Secret Sync Controller provides a unified way of pulling secrets from a cloud provider and storing them in the Kubernetes secret store for offline/intermittent access.
+
+## What is the benefit?
+
+1. Offline Access: Many clusters running in production environments need to be resilient to intermittent connectivity. This includes continuing to have access to secrets when a cluster goes offline and a pod restarts. Workloads can continue to access secrets by storing secrets in the Kubernetes secret store instead of a directly mounted volume (which is how the existing secret store CSI driver enables accessing secrets).
+
+2. Standard Kubernetes Secret Access: Secrets can be accessed via volume mounting, environment variables, or via the Kubernetes API. Workloads and ingress controllers do not need to be customized to access a cloud provider secret store and developers have options on how to access secrets.  
+
+3. Security: The Secret Sync Controller (SSC) has limited permissions and leverages the latest Kubernetes security features (complete list below) so that cluster admins do not need to configure and restrict permissions themselves.  
+
+## How to use the Secret Sync Controller
+
+The Secret Sync Controller (SSC) must be used alongside a [secret store Cloud Provider](https://secrets-store-csi-driver.sigs.k8s.io/getting-started/installation.html#install-external-secret-providers) that implements the Secret Store CSI Driver provider interface to provide an end-to-end solution for synchronizing secrets from the cloud secret store to the Kubernetes secret store.  
+
+1. Install the [secret sync controller](https://gist.github.com/aramase/46bd3d4270d9c44b59e8c2afe56fbc09) via a Helm chart, or YAML manifests alongside a secret store provider. If using the helm chart, users can configure parameters such as sync intervals and resource limitations at install time.
```

Comment:
Is there a more official installation reference other than a personal gist?

Reply:
We'd definitely prefer to link to something from the Kubernetes project than from a single contributor.

Reply:
The installation is currently in a gif because the work is in a PR and it's pre-alpha. We're in the process of moving the code to https://github.com/kubernetes-sigs/secrets-store-sync-controller and cutting the alpha release in the next few months. 

Would it still be fine referencing the gist given the above?